receiver/prometheusremotewritereceiver/receiver.go

```diff
@@ -340,26 +340,27 @@ func (prw *prometheusRemoteWriteReceiver) translateV2(_ context.Context, req *wr
 		metric, exists := metricCache[metricKey]
 		// If the metric does not exist, we create an empty metric and add it to the cache.
 		if !exists {
-			metric = scope.Metrics().AppendEmpty()
-			metric.SetName(metricName)
-			metric.SetUnit(unit)
-			metric.SetDescription(description)
-
```

Comment:
Makes more sense to set metric attributes for types that we have sure that will be handled. Otherwise we can just ignore them...